rockylinux/content.md

```diff
@@ -0,0 +1,85 @@
+# Rocky Linux
+
+Rocky Linux is a community-supported distribution derived from sources freely provided to the public by [Red Hat](ftp://ftp.redhat.com/pub/redhat/linux/enterprise/) for Red Hat Enterprise Linux (RHEL). As such, Rocky Linux aims to be functionally compatible with RHEL. The Rocky Linux Project mainly changes packages to remove upstream vendor branding and artwork. Rocky Linux is no-cost and free to redistribute. Each Rocky Linux version is maintained for up to 10 years (by means of security updates -- the duration of the support interval by Red Hat has varied over time with respect to Sources released). A new Rocky Linux version is released approximately every 2 years and each Rocky Linux version is periodically updated (roughly every 6 months) to support newer hardware. This results in a secure, low-maintenance, reliable, predictable, and reproducible Linux environment.
+
+Thank you for using Rocky Linux! We appreciate your feedback and welcome you to the community! Please join us on IRC
+on Libera.chat in #rockylinux or on our Mattermost instance at https://chat.rockylinux.org
+
+* [docs.rockylinux.org](https://docs.rockylinux.org)
+* [wiki.rockylinux.org](https://wiki.rockylinux.org)
+
+%%LOGO%%
+
+# Rocky Linux image documentation
+
+The `%%IMAGE%%:latest` tag is always the most recent version currently available.
+
+## Rolling builds
+
+Rocky Linux offers regularly updated images for all active releases. These images will be updated monthly or as needed for emergency fixes. These rolling updates are tagged with the major version number only. For example: `docker pull %%IMAGE%%:8`.
+
+## Minor tags
+
+Additionally, images with minor version tags that correspond to install media are also offered. **These images DO NOT receive updates** as they are intended to match installation iso contents. If you choose to use these images it is highly recommended that you include `RUN yum -y update && yum clean all` in your Dockerfile, or otherwise address any potential security concerns. To use these images, please specify the minor version tag:
+
+For example: `docker pull %%IMAGE%%:8.4`
+
+# Package documentation
+
+By default, the Rocky Linux containers are built using yum's `nodocs` option, which helps reduce the size of the image. If you install a package and discover files missing, please comment out the line `tsflags=nodocs` in `/etc/yum.conf` and reinstall your package.
+
+# Systemd integration
```

Comment:
This looks like it's a direct copy-pasta from `centos`?  You might want to check #1936 ... :grimacing:

(I'd suggest dropping this entire section either way.)

Reply:
Hm, it is indeed, but it does work in e.g., podman, which is mostly why I've included it.

I'm happy to remove it from the docker-library docs, or add a disclaimer that it doesn't work with docker, if that's the case. 

```
╭─<neil@afterman-ascension>-</tmp/neil/tmp.kHyNmPnTaP>-<9:45AM>-◇
╰─ podman exec -it 3f3134e735e2 systemctl restart httpd                                         130 ↵

╭─<neil@afterman-ascension>-</tmp/neil/tmp.kHyNmPnTaP>-<9:45AM>-◇
╰─ podman exec -it 3f3134e735e2 systemctl status httpd 
● httpd.service - The Apache HTTP Server
   Loaded: loaded (/usr/lib/systemd/system/httpd.service; enabled; vendor preset: disabled)
   Active: active (running) since Sat 2021-11-13 14:45:46 UTC; 1s ago
     Docs: man:httpd.service(8)
 Main PID: 251 (httpd)
   Status: "Started, listening on: port 80"
    Tasks: 213 (limit: 1638)
   Memory: 17.3M
   CGroup: /system.slice/httpd.service
           ├─251 /usr/sbin/httpd -DFOREGROUND
           ├─252 /usr/sbin/httpd -DFOREGROUND
           ├─253 /usr/sbin/httpd -DFOREGROUND
           ├─254 /usr/sbin/httpd -DFOREGROUND
           └─255 /usr/sbin/httpd -DFOREGROUND

Nov 13 14:45:46 3f3134e735e2 systemd[1]: Starting The Apache HTTP Server...
Nov 13 14:45:46 3f3134e735e2 httpd[251]: AH00558: httpd: Could not reliably determine the server's fu>
Nov 13 14:45:46 3f3134e735e2 systemd[1]: Started The Apache HTTP Server.
Nov 13 14:45:46 3f3134e735e2 httpd[251]: Server configured, listening on: port 80
```